### PR TITLE
Remove linting errors and some files cleaned up.

### PR DESCRIPTION
**Pull request by himanshuc3**

### CODE REVIEW

File: lib/spinner.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const helper = require('./utils/helper'),
```
Comment (Rishabh570) [inline lib/spinner.js]: A semicolon should be present at the end of this import.

**Review by Rishabh570 — COMMENTED** [review]
After that change, it is working fine on local. Btw consider rebasing after you're done :+1:

File: lib/spinner.js (hunk 1 above)
Comment (himanshuc3) [reply lib/spinner.js]: The latest commit was done post rebasing.

File: package.json [hunk 2]
```diff
@@ -4,11 +4,12 @@
     "description": "Live cricket scores served at your terminal 🏏",
     "main": "index.js",
     "scripts": {
-        "start": "node ./dist/index.js",
+        "start": "nodemon --exec npm run babel",
+        "babel": "babel-node index.js",
```
Comment (Rishabh570) [inline package.json:8]: I guess we should use the build file that parcel creates...What do you think?

Comment (himanshuc3) [reply package.json:8]: For the bundle, I agree, but in development, it doesn't allow to watch files. We would have to create a server using parcel config (which also uses babel config for esnext syntax and es6 modules if we want to use that in future). You could commit that on top of this branch. Other wise, I think this works fine right now.

Comment (Rishabh570) [reply package.json:8]: Yes, watching files is not really needed as of now. But, using the bundle gives some error during the build process. I guess we should go ahead with this for now and come back later for the bundling part. :+1: :rocket: 

Thanks!

**Review by Rishabh570 — APPROVED** [review]

**Pull request merged**